vault/mount.go

```diff
@@ -110,10 +132,17 @@ type MountEntry struct {
 	Type        string            `json:"type"`              // Logical backend Type
 	Description string            `json:"description"`       // User-provided description
 	UUID        string            `json:"uuid"`              // Barrier view UUID
-	Options     map[string]string `json:"options"`           // Backend configuration
+	Config      MountConfig       `json:"config"`            // Configuration related to this mount (but not backend-derived)
+	Options     map[string]string `json:"options"`           // Backend options
 	Tainted     bool              `json:"tainted,omitempty"` // Set as a Write-Ahead flag for unmount/remount
 }
 
+// MountConfig is used to hold settable options
+type MountConfig struct {
+	DefaultLeaseTTL *time.Duration `json:"default_lease_ttl" structs:"default_lease_ttl" mapstructure:"default_lease_ttl"` // Override for global default
```

Comment:
Yes -- because when these are passed around as part of a tune call, I have to know whether the values were unset, or set to 0 -- IOW, use the defaults. If the values are unset I don't want to change them, but the time.Duration zero value is 0.

I could have a separate struct for the http/api/command functions but that seemed unnecessary. When these are created within core, they are initialized to zero.